_dashboards/maps/selfhost-maps-server.md

```diff
@@ -80,7 +83,7 @@ Confirm the server is running by opening each of the following links in a browse
 * Map tiles URL: `http://localhost:8080/tiles/data/{z}/{x}/{y}.png`
 * Map tiles demo URL: `http://localhost:8080/`
 
-## Use the self-host maps server with OpenSearch Dashboards
+## Using the self-host maps server with OpenSearch Dashboards
```

Comment:
The current usage steps are used for existing Region map and Coordinate map in Visualization, we also need to update here for new Maps.

For new Maps, the steps can be found [here](https://github.com/opensearch-project/documentation-website/blob/52eb9431563f810575129ee4c74d0b1389051bff/_dashboards/maps/maps.md#adding-a-custom-map) with TMS type. The TMS URL from self-host server should be like: http://localhost:8080/tiles/data/{z}/{x}/{y}.png